Yoast/wordpress-seo
# Meta description for author archive

## Summary


This PR can be summarized in the following changelog entry:

* Adds a meta description presentation for author archives.

## Relevant technical choices:

* If a meta description is not present in the indexables table, take the meta description from the title options table.
    * Meta descriptions filled in in the user profile will end up in the indexables table, while meta descriptions filled in in the search appearance will end up in the title options table.

## Test instructions

This PR can be tested by following these 

* In the menu on the left, click `Users` --> `Your profile` --> `Meta description to use for Author page`
* Fill in a meta description, then click `Update profile`
* In the menu on the left, click `SEO` --> `Search appearance` --> `Archives` --> `Author archive settings`
* Fill in a meta description, then click `Save changes`
* Navigate to an author archive, for example `http://one.wordpress.test/author/admin/` (potentially replace `one` with the name of your installation)
* The `description` content is expected to be equal to what you entered on the user profile.
* In the menu on the left, click `Users` --> `Your profile` --> `Meta description to use for Author page`
* Remove the text from the title, then click `Update profile`
* The `description` content is expected to be equal to what you entered on the search appearance settings.
* In the menu on the left, click `SEO` --> `Search appearance` --> `Archives` --> `Author archive settings`
* Remove the text from the `Meta description`, then click `Save changes`
* The `description` content is expected to be empty

## UI changes

## Documentation

## Quality assurance


Fixes #
